src/AspNetCoreModuleV2/AspNetCore/src/applicationinfo.cpp

```diff
@@ -106,22 +106,16 @@ APPLICATION_INFO::UpdateAppOfflineFileHandle()
 
     ReferenceApplicationInfo();
 
-    if (INVALID_FILE_ATTRIBUTES == GetFileAttributes(strFilePath.QueryStr()) &&
-        GetLastError() == ERROR_FILE_NOT_FOUND)
+    if (INVALID_FILE_ATTRIBUTES == GetFileAttributes(strFilePath.QueryStr()))
```

Comment:
Should we check if GetLastError is FileNotFound or InvalidPath rather than removing this check?

Reply:
I looked through windows other source files and they don't generally check it. 
Additionally, what should we do if we get `INVALID_FILE_ATTRIBUTES` but some other LastError?